tests/functional/test_networks.py

```diff
@@ -146,6 +146,12 @@ def test_repr(networks_connected_to_tester):
     assert actual == expected
 
 
+def test_repr_disconnected(networks_disconnected):
+    assert repr(networks_disconnected) == "<NetworkManager>"
+    assert repr(networks_disconnected.ethereum) == "<ethereum>"
+    assert repr(networks_disconnected.ethereum.local) == "<local>"
```

Comment:
sort of wish this appeared like this
```suggestion
    assert repr(networks_disconnected.ethereum.local) == "<ethereum:local>"
```